package.json

```diff
@@ -30,7 +30,6 @@
 		"vscode.github-authentication"
 	],
 	"activationEvents": [
-		"*",
```

Comment:
Realized that this PR as it stands isn't enough, because GHPRI sets a lot of context keys on activation which hide commands, and all of those context keys probably need to be revisited to ensure that functionality isn't improperly hidden. @alexr00 maybe we can chat about this further when you are in next week?